dojo/forms.py

```diff
@@ -553,6 +553,9 @@ class TestForm(forms.ModelForm):
                            required=False,
                            help_text="Add tags that help describe this test.  "
                                      "Choose from the list or add new tags.  Press TAB key to add.")
+    lead = forms.ModelChoiceField(
+	queryset=User.objects.exclude(is_staff=False),
+	required=True, label="Testing Lead")
```

Comment:
Setting this field to required causes problems in some parts of the app, for instance when trying to delete a test you're first required to set a test lead before being allowed to delete it. It's probably fairly common as well to know that a test is coming up but no decision has been made about who will take the lead on it, so I think this field shouldn't be required unless there are good reasons that outweigh the mentioned issues.